src/main/java/com/Raisetech/Task7/original/DobController.java

```diff
@@ -0,0 +1,18 @@
+package com.Raisetech.Task7.original;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+@RestController
+public class DobController {
+
+ @GetMapping("/dobs")
+     public Map<String,String> getDobs(@RequestParam("date_of_birthday") String date_of_birthday) {
```

Comment:
生年月日ならば date of birth かと思います。
```suggestion
     public Map<String,String> getDobs(@RequestParam("date_of_birth") String date_of_birthday) {
```